README.rst

```diff
@@ -141,3 +141,27 @@ given severity and message::
             ('root', logging.INFO, 'boo arg'),
         ]
 
+
+Live Logs
+~~~~~~~~~
+
+By default, catchlog will output any logging records with a level higher or equal
+to WARNING. In order to actually see these logs in the console you have to disable
+pytest output capture by passing ``-s``. The life span of the CLI logging handler
+lasts throughout the whole runtests session.
```

Comment:
nitpick: I'd just say "the whole test session" here, as someone not familiar with pytest internals might not know what you mean with "runtests session".

Alternatively maybe get rid of the whole sentence? This sounds like an implementation detail which is irrelevant to a catchlog user anyways, no?